sli/parser.h

```diff
@@ -61,6 +61,7 @@ class Parser
 public:
   Parser();
   Parser( std::istream& );
+  ~Parser() { delete s; }
```

Comment:
There is another little formatting problem which I had overlooked yesterday. This should fix it:
```suggestion
  ~Parser()
  {
    delete s;
  }
```  
BTW, we have set up pre-commit checks for NEST to make it easier for contributors to submit code in accordance to our formatting guidelines, see https://nest-simulator.readthedocs.io/en/stable/developer_space/guidelines/coding_guidelines_check.html#tooling.

Reply:
Can you share why line 60 formatting is ok? I matched the destructor formatting to the constructor formatting.

Reply:
The destructor is defined inline, while the constructor is only declared in line 63, so there is no body, and the formatting problem related to the method body.